### PR TITLE
Merge to master 05 2020

### PR DESCRIPTION
**Pull request by oferShinaar**
merge bitbucket 01ba6a52815 to github/master

### CODE REVIEW

**Review by nidalf — APPROVED** [review]

**Review by ronen-haen-sndk — APPROVED** [review]

File: WD-Firmware/demo/build/demos/demo_external_interrupts.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    self.listDemoSpecificCFlags = [
+    ]
+
+    self.listDemoSpecificLinkerFlags = [
+      '-Wl,--defsym=__comrv_cache_size=' + self.strComrvCacheSize
+    ]
+    
+    self.listSupportedTargetBoards = [
+      'eh1'
+	]
```
Comment (nati-rapaport) [inline WD-Firmware/demo/build/demos/demo_external_interrupts.py:46]: Add new line here

File: WD-Firmware/demo/build/demos/demo_nmi.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    self.listDemoSpecificCFlags = [
+    ]
+
+    self.listDemoSpecificLinkerFlags = [
+      '-Wl,--defsym=__comrv_cache_size=' + self.strComrvCacheSize
+    ]
+    
+    self.listSupportedTargetBoards = [
+      'eh1'
+	]
```
Comment (nati-rapaport) [inline WD-Firmware/demo/build/demos/demo_nmi.py:46]: Add new line here

Comment (oferShinaar) [reply WD-Firmware/demo/build/demos/demo_nmi.py:46]: this should be done in the bitbucket. if this is critical then please create PR in BB

File: WD-Firmware/demo/build/demos/demo_external_interrupts.py (hunk 1 above)
Comment (oferShinaar) [reply WD-Firmware/demo/build/demos/demo_external_interrupts.py:46]: this should be done in the bitbucket. if this is critical then please create PR in BB

**Review by nati-rapaport — APPROVED** [review]

**Comment by oferShinaar** [comment]
Sync Github with latest work: 01ba6a52815

**Pull request merged**